website/docs/_advanced_install.mdx

```diff
@@ -330,6 +334,46 @@ sdk install scalacli
 </Tabs>
 
 </TabItem>
+
+<TabItem value="freebsd">
+<Tabs
+groupId="freebsd"
+defaultValue="manual"
+values={[
+{label: 'Manual', value: 'manual'},
+]}
+>
+
+<TabItem value="manual">
+FreeBSD support (and support for other BSDs) is currently very partial. Scala CLI does not distribute for FreeBSD, but you can install it via Coursier's standalone JAR.
+
+Before running Coursier and installing Scala CLI, make sure you have Java 17 or higher installed and in your $PATH:
+```bash
+pkg install openjdk17
+```
+Download Coursier via the [Any JAR option](https://get-coursier.io/docs/cli-installation#latest-launchers).
+
+After downloading Coursier, you can use it to install Scala CLI:
+```console
+chmod +x coursier
+./coursier install scala-cli
+```
+You now have scala-cli installed in your system. Scala CLI is installed as a JAR rather than a native binary, and so it has a slightly higher startup time.
```

Comment:
```suggestion
Scala CLI is now installed on your system. It is installed as a JAR rather than a native binary, and so the startup time is slightly longer.
```